### ISSUE TITLE
Internationalization with Spring Boot 2.1

### ISSUE THREAD

**Issue by arey** [spring-projects/spring-petclinic#363]
From Spring Boot 2.1 upgrade, I've noticed the Petclinic welcome page display a `??welcome_fr_FR??` message.

**Comment by snicoll** [comment]
Yep, indeed. I can see that the `MessageSource` is loaded properly so I wonder where that regression comes from.

**Comment by dsyer** [comment]
Works for me in Chrome with French as the default language.
